Examples/MagicWeather/lib/src/components/native_dialog.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'dart:io';
+
+class ShowDialogToDismiss extends StatelessWidget {
+  final String content;
+  final String title;
+  final String buttonText;
+
+  ShowDialogToDismiss({this.title, this.buttonText, this.content});
+
+  @override
+  Widget build(BuildContext context) {
+    if (!Platform.isIOS) {
+      return AlertDialog(
+        title: new Text(
+          title,
+        ),
+        content: new Text(
+          this.content,
+        ),
+        actions: <Widget>[
+          new FlatButton(
```

Comment:
There's a warning that says `FlatButton` is deprecated and `TextButton` should be used